scipy/stats/_morestats.py

```diff
@@ -3237,7 +3238,7 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False,
     return WilcoxonResult(T, prob)
 
 
-def median_test(*args, ties='below', correction=True, lambda_=1,
+def median_test(*samples, ties='below', correction=True, lambda_=1,
                 nan_policy='propagate'):
     """Perform a Mood's median test.
 
```

Comment:
Below, ``n = len(args)`` should be ``n = len(samples)`` now.

Reply:
Fixed